src/test/java/org/cloudfoundry/promregator/auth/OAuth2XSUAAEnricherTest.java

```diff
@@ -48,17 +54,12 @@ public static void cleanupEnvironment() {
 	@Test
 	void testAppropriateJWTCall() {
```

Comment:
From the pure doctrine point of view this tests with a mock server might not be required any more since we are testing the token-client which is a foreign dependency.
On the other hand this might still be appropriate as some kind of inbound qualification. Especially since there are some parts which are hard to unit test since it is related to mocking constructors which is hard to do via mockito (... at least to my knowledge).

Reply:
As discussed: I am not too religious about this aspect...